core/classifiers/base.py

```diff
@@ -53,12 +62,13 @@ def type_in_job_result_proto(self):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def to_dict(self):
-        """Returns a dict representing this classifier.
+    def to_proto(self):
+        """Returns a protobuf of the frozen model consisting of trained
+        parameters.
 
         Returns:
-            dict. A dictionary representation of classifier referred as
-            'classifier_data'. This data is used for prediction.
+            Object. A protobuf object of frozen model containing trained
```

Comment:
Just checking -- here and elsewhere, is "Object" the actual type? Figured I'd double-check since the name seems rather generic.

Perhaps worth changing the return type to "something.Object" if that helps disambiguate (again, here and elsewhere), based on where the type is defined.

Reply:
Using the `FrozenModel` definition as defined in [#10060](https://github.com/oppia/oppia/pull/10060). Also added README in `core/domain/proto`